AirCasting/SessionViews/StatisticsContainerView.swift

```diff
@@ -9,6 +9,7 @@ import SwiftUI
 
 struct StatisticsContainerView<ViewModelType>: View where ViewModelType: StatisticsContainerViewModelable {
     @ObservedObject var statsContainerViewModel: ViewModelType
+    @ObservedObject var threshold: SensorThreshold
```

Comment:
all things related to this (and this var itself too) should be moved to viewmodel since we're using one for this view. The viewmodel should be the only source of truth for the view and should decide about colors, label values, date formatting etc.

Reply:
Ok, so I tried, spent like 2 hours or more with different approaches (adding threshold/ thresholds, etc) and it always ends up in some dead end. 
So I decided to leave it as it is for now and switch to other essential tasks 😬